libvast/vast/system/source.hpp

```diff
@@ -126,6 +126,12 @@ struct source_state {
   measurement measurement_;
 
   void send_report() {
+    // Send the reader-specific status report to the accountant.
+    if (auto status = reader.status(); !status.empty()) {
+      if (accountant)
+        self->send(accountant, std::move(status));
+    }
```

Comment:
```suggestion
    if (auto status = reader.status(); !status.empty())
      if (accountant)
        self->send(accountant, std::move(status));
```

Optional: omit the extra braces.

Reply:
```suggestion
    if (auto status = reader.status(); !status.empty() && accountant) {
      self->send(accountant, std::move(status));
    }
```